modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/common/components/Breadcrumb.tsx

```diff
@@ -29,6 +45,69 @@ export interface BreadcrumbItem {
 	onClick?: () => void;
 }
 
+function ActionDropdownItem({
+	confirmationMessage,
+	href = '',
+	label,
+	redirect,
+	size = 'full-screen',
+	target = 'link',
+	...props
+}: {label: string} & ActionDropdownItemProps) {
+	const handleTargetAction = async (event: any) => {
+		if (target === 'modal') {
+			event.preventDefault();
```

Comment:
just wondering: are these `event.preventDefault()` really needed? If it is, what if we prevent it first thing on `handleClick()`